hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RegionCoprocessorHost.java

```diff
@@ -102,6 +102,11 @@
   // optimization: no need to call postScannerFilterRow, if no coprocessor implements it
   private final boolean hasCustomPostScannerFilterRow;
 
+  @VisibleForTesting
```

Comment:
OK, VisibleForTesting...

We have a dicussion thread in the mailing list, and @apurtell concluded that we do not use annotation at all, just use a javadoc, and had already done a PR to remove all the VisibleForTesting annotations from our code base. I'm still trying to convince him to allow this annotation on IA.Private classes.

There is still no consensus yet, you can see the discussion in this PR
https://github.com/apache/hbase/pull/2714

So here I suggest that we remove this annotation to align with the current rule in our code base first. But anyway, I think this is an example that developers want to use this annotation, so maybe @apurtell could change his opinion.

Thanks.

Reply:
Yeah, VisibleForTesting is useless. Will remove this.